src/SplitPane.js

```diff
@@ -187,6 +187,9 @@ class SplitPane extends React.Component {
       allowResize,
       children,
       className,
+      paneClassName,
+      pane1ClassName,
+      pane2ClassName,
```

Comment:
Can you add the prop types for these as well please, also we're ordering these alphabetically...